src/features/place-bid/PlaceBidForm/hooks/usePlaceBidForm.tsx

```diff
@@ -88,32 +89,36 @@ export const usePlaceBidForm = (zna: string): UsePlaceBidFormReturn => {
 		);
 	};
 
-	const onConfirmPlaceBid = () => {
+	const onConfirmPlaceBid = async () => {
```

Comment:
I was unable to use `useTransaction` hook here as place bid doesn't use 'tx.wait`, therefore `wait` is undefined and throws an error.